### ISSUE TITLE
Confusion about feature maps/ heatmaps

### ISSUE THREAD

**Issue by lschmidtke** [facebookresearch/PoseWarper#10]
Hi,

first of all thanks for providing the code. While reading the paper and looking through your code, I got a bit confused with the following: In your paper you state that you compute the difference between the two heatmaps, feed them though some conv layers to receive offsets, and use these to warp heatmap B. This seems to happen in the code as well. But figure 2 implies that you are computing the difference between two feature encodings (from the last layer of the downsampling path), then compute the offsets, warp the encoding and then decode it back into heatmaps. Would be great to get some clarification here.

**Comment by gberta** [comment]
Figure 2 in the paper is made for illustrative purposes. In practice, we can do the warping either on top of pose heatmaps or on top of any of the selected features. In our experiments, we tried both schemes, and observed that warping on top of heatmaps worked slightly better in the context of video pose estimation. Therefore, in the end we used the scheme where the difference is computed between two heatmaps, which is then used to predict the offsets. Lastly, these offsets are used to warp pose heatmap from one frame to another. Hope this makes it clearer, and sorry for the confusion!
